query/src/main/scala/filodb/query/exec/AggrOverRangeVectors.scala

```diff
@@ -703,6 +770,7 @@ class TopBottomKRowAggregator(k: Int, bottomK: Boolean) extends RowAggregator {
   }
 }
 
+//scalastyle:off
```

Comment:
Is this necessary?

Reply:
The reason to add this is to avoid 800line limit for a single file.